RUNNING_INFERSHARP_ON_WINDOWS.md

```diff
@@ -0,0 +1,40 @@
+# How to Run Infer# Locally on Windows via WSL2 (Windows Subsystem for Linux)
+
+## Prerequisite
+- [Enable WSL2](https://docs.microsoft.com/en-us/windows/wsl/install-win10).
+*We recommend using Ubuntu or Debian WSL distribution.*
+
+## Setup
+1. Download the Infer# binaries from the [latest release page](https://github.com/microsoft/infersharp/releases).
+2. From Windows Command Prompt or PowerShell, enter: ```wsl.exe``` to open your default Linux distribution.
+3. Copy the _infersharp_ folder to _/home/<YOUR_USERNAME>_ and create a symlink for future use.
+**Important** - Because of a known performance issue on slow I/O between Windows and Linux file systems, please make sure you copy the binaries to only Linux file system.
```

Comment:
say: **Important** - Because of a known performance issue on slow I/O between Windows and Linux file systems, copy the binaries only to Linux.

Though, why are we saying this anyway? Is the default to copy to both file systems at once or something?

Reply:
You can think of Linux has its own file system in WSL. This is to make sure that if users want to copy the binaries to a different location other than /home/, they should choose a location on the Linux file system, not Windows (e.g. not C:\infersharp\...).

Reply:
Makes sense -- just the wording then